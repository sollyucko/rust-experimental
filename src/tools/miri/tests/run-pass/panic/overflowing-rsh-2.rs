// ignore-windows: Unwind panicking does not currently work on Windows
#![allow(arithmetic_overflow)]

fn main() {
    // Make sure we catch overflows that would be hidden by first casting the RHS to u32
    let _n = 1i64 >> (u32::MAX as i64 + 1);
}
