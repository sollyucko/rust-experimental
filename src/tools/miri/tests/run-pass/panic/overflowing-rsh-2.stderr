thread 'main' panicked at 'attempt to shift right with overflow', $DIR/overflowing-rsh-2.rs:6:14
